tests/DealsServiceTest.php

```diff
@@ -63,6 +63,14 @@ public function testUpdate()
  
   }
 
+  public function testUpdateWithoutProvidingDealValue()
+  {
+      unset(self::$deal['value']);
+      $updatedDeal = self::$client->deals->update(self::$deal['id'], self::$deal);
+      $this->assertInternalType('array', $updatedDeal);
+      $this->assertEquals($updatedDeal['id'], self::$deal['id']);
```

Comment:
I'd check here deal's whether value has not been changed

Reply:
btw: what about other properties? does it work when given property is not set? especially what about decimal value? 

what was the reason for this change?

Reply:
Added checking that value hasn't changed